src/scripts/components/MainDashboard.jsx

```diff
@@ -0,0 +1,193 @@
+/**
+ * @jsx React.DOM
+ */
+
+'use strict';
+
+var React = require('react/addons'),
+    Router = require('react-router'),
+	Button = React.createFactory(require('./Button.jsx')),
+	WidgetBoxes = React.createFactory(require('./WidgetBoxes.jsx')),
+	EditableModal = React.createFactory(require('./EditableModal.jsx')),
+	LogoutMessage = React.createFactory(require('./LogoutMessage.jsx'));
+
```

Comment:
Since `WidgetBoxes` is a ReactComponent and we are using JSX here, I think `var WidgetBoxes = require('./WidgetBoxes.jsx')` should work. Why do we need to call `React.createFactory` instead here ?